src/sentry/api/endpoints/project_users.py

```diff
@@ -30,7 +30,7 @@ def get(self, request, project):
                 return Response([])
             try:
                 queryset = queryset.filter(
-                    **{"{}__icontains".format(EventUser.attr_from_keyword(pieces[0])): pieces[1]}
+                    **{f"{EventUser.attr_from_keyword(pieces[0])}__icontains": pieces[1]}
```

Comment:
The downside to automated f-string conversions is that changes like these aren't very readable anymore, and instead should be refactored to pull things out into more meaningful variable names.

Reply:
A more extreme example of this is what I mentioned in the PR description, https://github.com/getsentry/sentry/pull/23933/commits/35d4d8f753c0d2623467fd6d1018546351ff3e6a, which I refactored after an initial automated pass.